### DISCUSSION TITLE
Suggestion: Make a Keepass2Android Plugin that allows sending credentials directly from the password manager to the connected device

### DISCUSSION THREAD

**Discussion by jpo234** [AppGround-io/bluetooth-keyboard-and-mouse-support#191] [Ideas]
The popular password manager [Keepass2Android](https://github.com/PhilippC/keepass2android) has a plugin system that allows sending the credentials somewhere else. An example is the [KP2A InputStick Plugin](https://github.com/inputstick/KP2APlugin), It requires a usb hardware receiver that emulates an USB keyboard (see http://inputstick.com/).
It would be really useful to have a similar plugin that does away with the need for a hardware usb receiver and uses your app instead.

**Comment by phedders** [comment]
Definite thumbs up for this one....

**Comment by Kaptensvart** [comment]
Cool idea 💡

**Comment by ianmperryman** [comment]
Cool idea, but not sure you want to send passwords on Bluetooth.  
https://vpnoverview.com/privacy/devices/bluetooth/
